Change default config file to better fit the debian policy for file location.
--- a/etc/pyca.conf
+++ b/etc/pyca.conf
@@ -38,7 +38,7 @@
 # history of recordings. This can be any database supported by SQLAlchemy.
 # Type: String
 # Default: sqlite:///pyca.db
-#database = sqlite:///pyca.db
+database = sqlite:////var/lib/pyca/pyca.db
 
 
 [capture]
@@ -47,7 +47,7 @@
 # will be created.
 # Type: string
 # Default: ./recordings
-directory        = './recordings'
+directory        = '/var/lib/pyca/recordings'
 
 # Command to use for capturing. This may also be a script file. The only
 # requirement is that the command terminates itself in time.
@@ -93,7 +93,7 @@
 # be as confidence monitoring in the web ui.
 # Type: String
 # Default: ./recordings
-#preview_dir      = './recordings'
+#preview_dir      = '/var/lib/pyca/recordings'
 
 # List of preview images to be included in the web ui.
 # Type: list of strings (write as '...', '...')
